### PR TITLE
add compat data for MediaSettingsRange

### PR DESCRIPTION
**Pull request by gsouquet**
https://developer.mozilla.org/en-US/docs/Web/API/MediaSettingsRange

### CODE REVIEW

File: api/MediaSettingsRange.json [hunk 1]
```diff
@@ -0,0 +1,196 @@
...
+          "safari": {
+            "version_added": null
+          },
+          "safari_ios": {
+            "version_added": null
+          }
+        },
+        "status": {
+          "experimental": true,
+          "standard_track": false,
```
Comment (ddbeck) [inline api/MediaSettingsRange.json]: Looks to be [`true`](https://w3c.github.io/mediacapture-image/#mediasettingsrange-section)

File: api/MediaSettingsRange.json [hunk 2]
```diff
@@ -0,0 +1,196 @@
...
+            "safari": {
+              "version_added": null
+            },
+            "safari_ios": {
+              "version_added": null
+            }
+          },
+          "status": {
+            "experimental": true,
+            "standard_track": false,
```
Comment (ddbeck) [inline api/MediaSettingsRange.json]: Also `true`

File: api/MediaSettingsRange.json [hunk 3]
```diff
@@ -0,0 +1,196 @@
...
```
Comment (ddbeck) [inline api/MediaSettingsRange.json]: Also `true`

File: api/MediaSettingsRange.json [hunk 4]
```diff
@@ -0,0 +1,196 @@
...
```
Comment (ddbeck) [inline api/MediaSettingsRange.json]: Also `true`

**Comment by gsouquet** [comment]
All fixed! Thanks @ddbeck for the review

**Comment by ddbeck** [comment]
Looks great. Thanks for the updates, @gsouquet!

**Pull request merged**